### DISCUSSION TITLE
Is it possible to make features in Rich Text field specific for certain field?

### DISCUSSION THREAD

**Discussion by alexanderkalachev** [payloadcms/payload#7567] [Feature Requests & Ideas]
Like in Sanity, you can create a specific configuration of their Rich Text alternative – Portable Text and use it in certain field.
For example, to have one version of Rich Text for page content with all the blocks, etc...
And the other with limited text styling and links.

**Comment by andershermansen** [comment]
Yes, you can override the editor to use for each individual field. 

See: https://payloadcms.com/docs/fields/rich-text
and https://payloadcms.com/docs/rich-text/lexical

So by passing a different editor instance you can have different features for different fields.

**Reply by alexanderkalachev** [reply]
Thank you! I haven't noticed this part in documentation previously.
